src/__stories__/chip-story.tsx

```diff
@@ -35,29 +37,32 @@ type Props = {
 const ChipBackgroundContainer = ({inverse, dataAttributes, children}: Props) => (
     <ResponsiveLayout isInverse={inverse} fullWidth>
         <Box padding={16} width="fit-content" dataAttributes={dataAttributes}>
-            <div
-                style={{
-                    // prevent line-height from affecting the height of the container;
-                    // happens when changing the base font size
-                    lineHeight: 0,
-                }}
-            >
-                {children}
-            </div>
+            <div style={{lineHeight: 0}}>{children}</div>
         </Box>
     </ResponsiveLayout>
 );
 
-export const Default: StoryComponent<Args> = ({inverse, icon, closable, badge}) => {
+export const Default: StoryComponent<Args> = ({
+    inverse,
+    icon,
+    closable,
+    badge,
+    active: chipActive,
+    href: hrefProp,
+}) => {
```

Comment:
instead of overloading this story, I'd create a new for touchable chips, where you can enable/disable the `active` prop. The same way we already have different stories for single/multiple selection chips
![image](https://github.com/user-attachments/assets/2e8324e4-ff7f-43c8-94bc-6884cf4bfd07)